.github/workflows/deployment.yml

```
@@ -19,6 +19,11 @@ jobs:
           echo "$SSH_KEY" > ~/.ssh/ssh_key
           chmod 600 ~/.ssh/ssh_key
 
+      - name: Ensure remote app directory exists
+        run: |
+          ssh -i ~/.ssh/ssh_key -o StrictHostKeyChecking=no ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST }} "mkdir -p ~/app"
+
+      - name: Transfer docker-compose-prod.yml to server
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)